app/javascript/onboarding/components/FollowUsers.jsx

```diff
@@ -136,10 +146,11 @@ class FollowUsers extends Component {
         </div>
         <div className="onboarding-selection-status">
           <div className="selection-status-content">
+            <p>{followingStatus}</p>
             <button type="button" onClick={() => this.handleSelectAll()}>
-              Select All 
-              {' '}
-              {selectedUsers.length === users.length ? '✅' : ''}
+              {selectedUsers.length === users.length
+                ? 'Deselect all'
```

Comment:
This logic is missing a specific situation where there are no users to begin with. Here's what displays in that case:
<img width="830" alt="Screen Shot 2020-04-14 at 8 56 27 AM" src="https://user-images.githubusercontent.com/6921610/79246177-f529b200-7e2d-11ea-9aab-f06730533593.png">

I think we should account for this because, as #7106 suggests, some new communities won't have users to follow by default.

Reply:
I updated it so that it will render no toggle button if there are no users to follow